frontend/src/pages/dashboard/DashboardPageView.jsx

```diff
@@ -0,0 +1,42 @@
+import { Card, Fab, styled } from '@mui/material'
+import AddIcon from '@mui/icons-material/Add'
+
+import HeaderCustom from '../../components/layout/headercustom/HeaderCustomController'
+import Footer from '../../components/layout/footer/FooterController'
+import Post from '../../components/posts/post/PostController'
+
+/**
+ * Card container for each feed post
+ */
+const FeedCard = styled(Card)({
+    width: '90%',
+    margin: 'auto',
+    marginTop: '10px',
+    marginBottom: '10px',
+    padding: '0 10px 0 10px',
```

Comment:
The cards look a bit cramped with no vertical padding, there should be some space between the interactions and the bottom of the card 

current:
![image](https://user-images.githubusercontent.com/54062686/158168001-ad7f453e-03c9-4ce3-b48c-cbe90398fb2a.png)

example with 10px 10px 10px 10px padding
![image](https://user-images.githubusercontent.com/54062686/158168056-9ca0fde0-818b-4de3-bc89-89817602a400.png)

Reply:
Will add some padding to post cards